services/executionengine/src/main/java/tech/pegasys/teku/services/executionengine/ExecutionEngineConfiguration.java

```diff
@@ -59,12 +70,13 @@ public Version getVersion() {
     private Spec spec;
     private Optional<String> endpoint = Optional.empty();
     private Version version = Version.DEFAULT_VERSION;
+    private Optional<String> jwtSecretFile;
```

Comment:
Need to provide a default value here:
```suggestion
    private Optional<String> jwtSecretFile = Optional.empty();
```